packages/autocomplete-theme-classic/src/theme.scss

```diff
@@ -1,33 +1,70 @@
 // ----------------
-// Variables
+// 1.  CSS Variables
+// 2.  Dark Mode
+// 3.  Autocomplete
+// 4.  Panel
+// 5.  Sources
+// 6.  Hit Layout
+// 7.  Panel Header
+// 8.  Panel Footer
+// 9.  Detached Mode
+// 10. Gradients
+// ----------------
+
+// Note:
+// This theme is trying to reflect the markup of autocomplete with SCSS indentation.
+// We use the SASS `@at-root` function to remove unecessary specificities.
+
+// ----------------
+// 1. CSS Variables
 // ----------------
 :root {
+  // input
+  --aa-search-input-height: 44px;
+  --aa-input-icon-size: 20px;
+  // size & spacing
   --aa-base-unit: 16;
-  --aa-font-size: calc(var(--aa-base-unit) * 1px);
-  --aa-spacing-factor: 1; // xs:0.6 / sm:0.8 / md:1 / lg:1.2 / xl:1.5
+  --aa-spacing-factor: 0.8;
   --aa-spacing: calc(var(--aa-base-unit) * var(--aa-spacing-factor) * 1px);
   --aa-spacing-half: calc(var(--aa-spacing) / 2);
+  --aa-panel-max-height: 650px;
+  // font
+  --aa-font-size: calc(var(--aa-base-unit) * 1px);
+  --aa-font-family: -apple-system, 'BlinkMacSystemFont', 'Segoe UI', 'Roboto',
```

Comment:
should we not inherit the document's font?

Reply:
I think it's unlikely a site doesn't have any font set, better to set those default styles in the example imo

Reply:
I like the idea of looking native by default, also every codesandboxes we created without that rule didn't looked nice by default. Motivations behind this doesn't seem strong, I'm open.

Reply:
@Shipow That's because you'd have to set the `font-family` manually everywhere right?

Reply:
What's the decision on this? In a nutshell we have two options:

1. Set `--aa-font-family` to `inherit` which will set it to what the user has set in `html` or `body`. If they want to change it, they must override the variable. On our end, we'll need to update all demos to set the `font-family` on the `html` or `body` to the current font stack.
2. Leave it as is. If users want to change it, they must override the variable.

In both cases, for the user, changing the font stack means overriding the variable. However, there's a bigger chance they will have to do it in scenario number 2.

Reply:
guess it's safe to go 1